docs/framework/data/adonet/sql/server-and-database-roles-in-sql-server.md

```diff
@@ -52,7 +52,7 @@ All versions of SQL Server use role-based security, which allows you to assign p
   
 |Resource|Description|  
 |--------------|-----------------|  
-|[Identity and Access Control](https://msdn.microsoft.com/library/bb510418.aspx) in SQL Server Books Online|Contains links to topics that describe principals, roles, credentials, securables and permissions.|  
+|[Identity and Access Control (Database Engine)](https://docs.microsoft.com/previous-versions/sql/sql-server-2008-r2/bb510418(v=sql.105)) in SQL Server Books Online|Contains links to topics that describe principals, roles, credentials, securables and permissions.|  
```

Comment:
probably best to change this to https://docs.microsoft.com/en-us/sql/relational-databases/security/authentication-access/getting-started-with-database-engine-permissions which is where those same child topics are now listed